### PR TITLE
adding support for handling mersi3 data

### PR DESCRIPTION
**Pull request by BengtRydberg**
Adding support for handling mersi3 data by renaming and extending the mersi2 script to also handle mersi3 data.
 
 - [x] Tests added 
 - [x] Tests passed: Passes ``pytest level1c4pps`` 
 - [x] Passes ``flake8`` 
 - [ ] Fully documented (don't really know what is needed)

### CODE REVIEW

File: setup.py
Comment (ninahakansson) [inline setup.py:1]: Maybe we need to keep the possibility to also run mersi22pps.py until next release of PPS.

Comment (BengtRydberg) [reply setup.py:1]: ok, I see. Do you know if it is possible to fix this through the setup.py file, so we don't need duplicated files? If one use entry_points in setup.py, this should be possible, but the bin directory is not a package so a bit tricky.  Or should I just keep the old mersi22pps.py file?

Comment (BengtRydberg) [reply setup.py:1]: I added mersi22pps.py as a symbolic link to mersi2pps.py. Is this an acceptable solution?

Comment (ninahakansson) [reply setup.py:1]: That will be fine!

**Pull request merged**